.github/workflows/sync_docs.yaml

```diff
@@ -1,33 +1,18 @@
-name: Sync docs from Discourse
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync Discourse docs
 
 on:
   workflow_dispatch:
   schedule:
-    - cron: '53 0 * * *' # Daily at 00:53 UTC
-  push:
-    branches:
-      - main
+    - cron: '53 0 * * *'  # Daily at 00:53 UTC
 
 jobs:
   sync-docs:
-    name: Open PR with docs changes
-    runs-on: ubuntu-latest
+    name: Sync docs from Discourse
+    uses: canonical/data-platform-workflows/.github/workflows/sync_docs.yaml@download-discourse-topics
```

Comment:
Add a `TODO` comment to keep a reminder

Reply:
Yeah well noticed - I'll actually go the proper route and change this to the (future) merged version instead of a branch. Will keep the PR on hold until then.